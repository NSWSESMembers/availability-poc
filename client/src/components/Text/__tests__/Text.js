import React from 'react';
import renderer from 'react-test-renderer';

import { Text } from '../';

test('renders correctly', () => {
  const tree = renderer.create(
    <Text>Test</Text>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
