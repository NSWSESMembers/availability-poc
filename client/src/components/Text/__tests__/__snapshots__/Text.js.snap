// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#6B6B6B",
        "fontSize": 16,
      },
      undefined,
      Object {},
    ]
  }
>
  Test
</Text>
`;
